src/Compiler/Microsoft.AspNetCore.Razor.Language/test/Legacy/HtmlMarkupParserTests.cs

```diff
@@ -203,9 +203,10 @@ public TestHtmlMarkupParser(ParserContext context)
             EnsureCurrent();
         }
 
-        public void Dispose()
+        void IDisposable.Dispose()
```

Comment:
Why is this needed?

Reply:
Because I implemented `IDisposable` on the base class, which conflicts with this derived implementation.